### ISSUE TITLE
Training set

### ISSUE THREAD

**Issue by InstantWindy** [KaiyangZhou/deep-person-reid#380]
Hi. What are the training settings for these models(mlfn,hacnn,osnet_x1_0,resnet50_fc512 )? To reproduce your test results. Thanks.

**Comment by KaiyangZhou** [comment]
are you referring to the models on the model zoo?

if so, the training params can be found from the file name

**Comment by InstantWindy** [comment]
I mean that I want to train these models(mlfn,hacnn,osnet_x1_0,resnet50_fc512) from scratch, but how to set the learning rate, epoch, etc. for training these models.

**Comment by KaiyangZhou** [comment]
you can follow the training params stated in this paper https://arxiv.org/abs/1905.00953 (don't use `random_patch` because it will cause a model to crash in cross-domain datasets)

**Comment by InstantWindy** [comment]
Thank for your reply. but In this paper, there are only osnet training settings.  How do you set the training epoch of other models and the learning rate to get the [same-domain](https://kaiyangzhou.github.io/deep-person-reid/MODEL_ZOO.html) result.

**Comment by KaiyangZhou** [comment]
oh, I see

for resnet and its variants like mlfn, use https://github.com/KaiyangZhou/deep-person-reid/blob/master/configs/im_r50_softmax_256x128_amsgrad.yaml

for other models, please search over the issues (using keywords) where you might find the answers

**Comment by InstantWindy** [comment]
Thank for your reply

**Comment by InstantWindy** [comment]
Thanks.
